src/core/GizmoHelper.tsx

```diff
@@ -115,8 +118,23 @@ export const GizmoHelper = ({
           mainCamera.position.set(0, 0, 1).applyQuaternion(q1).multiplyScalar(radius.current).add(focusPoint.current)
           mainCamera.up.set(0, 1, 0).applyQuaternion(q1).normalize()
           mainCamera.quaternion.copy(q1)
-          if (onUpdate) onUpdate()
-          else if (defaultControls) defaultControls.update()
+          if (onUpdate) {
+            onUpdate()
+          } else if (defaultControls) {
+            if (defaultControls instanceof CameraControls) {
+              defaultControls.updateCameraUp()
+              defaultControls.setLookAt(
+                mainCamera.position.x,
+                mainCamera.position.y,
+                mainCamera.position.z,
+                focusPoint.current.x,
+                focusPoint.current.y,
+                focusPoint.current.z
+              )
+            } else if (isOrbitControls(defaultControls)) {
```

Comment:
this imo would now tie it to either cc or oc, but all other controls would break